### PR TITLE
Adds volume control to admin-played sounds

### PR DESCRIPTION
**Pull request by ghost**
Having messed around with .oggs and the like when running (_yes I got admins to approve it >.>_) events, I figured this would be a nice thing to have ~~and would let Kev spam his midis at a volume that isn't deafening~~.

### CODE REVIEW

**Review by Cyberboss — APPROVED** [review]

**Comment by lzimann** [comment]
An extra input for this is annoying, admins can just figure out a non-shit volume when editing the sound.

**Comment by GunHog** [comment]
Because @kevinz000 needs the ability to play everything at max volume! :+1:

File: code/modules/admin/verbs/playsound.dm [hunk 1]
```diff
@@ -9,6 +9,11 @@
 		freq = pick(0.5, 0.7, 0.8, 0.85, 0.9, 0.95, 1.1, 1.2, 1.4, 1.6, 2.0, 2.5)
 		to_chat(src, "You feel the Honkmother messing with your song...")
 
+	var/vol = input(usr, "What volume would you like the sound to play at?",,) as null|num
```
Comment (lzimann) [inline code/modules/admin/verbs/playsound.dm]: at the very least make the default value to be 100

**Review by lzimann — CHANGES_REQUESTED** [review]
Input's default needs to be 100

**Review by lzimann — APPROVED** [review]

**Comment by MrStonedOne** [comment]
volume doesn't work on midis

This was tried before and broke everything.

when testing, please confirm that enabling admin sounds after one has played works correctly, and ensure that disabling admins sounds while one is playing works correctly.

**Comment by Xhuis** [comment]
Volume *does* seem to work with midis. Testing seems to work fine; I can also queue up sounds of differing volumes and they'll all play fine. It seems to be completely fine, but if you think it's too iffy then feel free to do some testmerging just to make sure everything runs smoothly.

**Comment by MrStonedOne** [comment]
when testing, please confirm that enabling admin sounds after one has played works correctly, and ensure that disabling admins sounds while one is playing works correctly.

**Comment by MrStonedOne** [comment]
when testing, please confirm that enabling admin sounds after one has played works correctly

**Comment by MrStonedOne** [comment]
when testing, please confirm that disabling admins sounds while one is playing works correctly.

**Comment by Xhuis** [comment]
Those all work, yes.

**Comment by MrStonedOne** [comment]
Ｃｏｏｌ

**Comment by Xhuis** [comment]
I'm the coolest coder here. 😎

**Pull request merged**